libs/shared/flagd-core/src/lib/flagd-core.ts

```diff
@@ -91,35 +92,43 @@ export class FlagdCore {
       }
     }
 
-    const defaultVariant = flag.defaultVariant;
-
-    let resolvedVariant;
+    let variant;
     let reason;
 
-    if (flag.targetingString == undefined) {
-      resolvedVariant = flag.variants.get(defaultVariant);
+    if (flag.targeting == undefined) {
```

Comment:
```suggestion
    if (!flag.targeting) {
```

Mostly this is how people do this sort of check from a style perspective, but also keep in mind `null !== undefined`. Note that`null == undefined` though, so this probably works as you expect... you just tend to see the style above much more.

Reply:
This is done for all usages hence will resolve this convo